karate-core/src/main/java/com/intuit/karate/Main.java

```diff
@@ -66,9 +67,15 @@
     @Parameters(split = "$", description = "one or more tests (features) or search-paths to run")
     List<String> paths;
 
-    @Option(names = {"-m", "--mock"}, description = "mock server file")
+    @Option(names = {"--mock"}, description = "mock server file")
     File mock;
 
+    @Option(names = {"-m", "--mocks"}, split = ",", description = "one or more mock server files")
+    List<File> mocks;
+
+    @Option(names = {"-P", "--prefix"}, description = "mock server prefix (contextPath)")
```

Comment:
this is good. can you update the description to `mock server path prefix (context-path)`